src/components/Chart.tsx

```diff
@@ -0,0 +1,156 @@
+/** Load dependencies */
+import React from "react";
+import { Chart as ChartJS } from "chart.js";
+import "chartjs-plugin-datalabels";
+
+/** Load types */
+import { Metric } from "../types";
+
+/** Types */
+type Context =
+  | string
+  | CanvasRenderingContext2D
+  | HTMLCanvasElement
+  | ArrayLike<CanvasRenderingContext2D | HTMLCanvasElement>;
+
+type RefContext =
+  | string
+  | ((instance: HTMLCanvasElement | null) => void)
+  | React.RefObject<HTMLCanvasElement>
+  | null
+  | undefined;
+
+export type ChartTypes =
+  | "line"
+  | "bar"
+  | "radar"
+  | "pie"
+  | "doughnut"
+  | "polarArea";
+
+/** Interfaces */
+interface Datalabels {
+  anchor: string;
+  align: string;
+  formatter: (x: number) => number;
+  font: {
+    weight: string;
+  };
+}
+
+interface ChartProps {
+  className?: string;
+  data: {
+    labels: Metric["label"][];
+    values: Metric["value"][];
+  };
+  types?: ChartTypes[];
+}
+
+/** Component */
+const Chart: React.FunctionComponent<ChartProps> = ({
+  className,
+  data,
+  types,
+}: ChartProps) => {
+  const [chartType, setChartType] = React.useState<ChartTypes>("bar");
+  const ctx: React.MutableRefObject<Context> = React.useRef<Context>("");
```

Comment:
```suggestion
  const ctx: React.MutableRefObject<Context> = React.useRef<Context>("");
```

Do you think that `React.MutableRefObject<Context>` is necessary?  Or it can be rewriten?